.github/workflows/unit.yaml

```diff
@@ -1,11 +1,13 @@
-name: Unit tests
+name: CI
 
 on:
   push:
     branches: [ master ]
+    tags: 
+      - '*'
   pull_request:
     branches: [ master ]
-
+      
```

Comment:
I think the reason `git` (the command line version, anyway) marks excess whitespace in red is to avoid situations like this, where there is no real change but the line is recorded as having been changed.